.changeset/four-wolves-tease.md

```diff
@@ -0,0 +1,5 @@
+---
+'@openfn/cli': minor
```

Comment:
I'd probably call this a patch actually @midigofrank  - it doesn't change the external API. Let's treat it as a fix to the previous.

I'll go ahead and do this now